docs/Troubleshooting.md

```diff
@@ -111,19 +111,32 @@ reasons. To expedite resolution of stack deployment issues, it's helpful to
 have the following data and output from the following [AWS CLI](https://aws.amazon.com/cli/) 
 commands when contacting support@quiltdata.io.
 
-1. Installed Quilt stack template metadata: This is located in the
-CloudFormation Quilt stack Outputs tab with the Key `TemplateBuildMetadata`
-and is a JSON object generated by the Quilt build system.
-
-![TemplateBuildMetadata Screenshot](imgs/cloudformation-stack-metadata.png)
-
-2. Quilt stack drift:
-
-    `$ aws cloudformation describe-stack-resource-drifts --stack-name QUILT_STACK_NAME`
+1. Quilt stack outputs:
+    <!--pytest.mark.skip-->
+    ```sh
+    STACK_NAME="YOUR_QUILT_STACK"
+    AWS_REGION="YOUR_REGION"
```

Comment:
1. I think both stack name and region vars don't have to be quoted, but that's OK
2. I'm not sure we should mention regions here because that seems to be a part of basic stuff like we're assuming they have credentials and we don't mention `--profile` option
3. it looks like `--region` would be unnecessary if you use `AWS_DEFAULT_REGION` instead of `AWS_REGION`